src/logic/getLibcore.js

```diff
@@ -0,0 +1,72 @@
+import { Platform, NativeModules } from "react-native";
+
+let core;
+let walletPoolInstance;
+let threadDispatcher;
+
+export default async () => {
+  if (core) {
+    return core;
+  }
+
+  if (Platform.OS === "android") {
```

Comment:
(future notes. we can probably have split into two files getLibcore.ios.js and getLibcore.android.js )

Reply:
I thought of it, but as there is a bit of common logic (instanciating wallet pool & pal) I chose to go like that